compiler/rustc_target/src/spec/targets/m68k_unknown_linux_gnu.rs

```diff
@@ -12,11 +12,18 @@ pub(crate) fn target() -> Target {
             description: Some("Motorola 680x0 Linux".into()),
             tier: Some(3),
             host_tools: Some(false),
-            std: None, // ?
+            std: Some(false), // std cannot be linked for now
```

Comment:
I think this should be a separate commit as it's an unrelated change.

Reply:
done